Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -519,6 +527,7 @@ - (void)rewardedAdDidFailToPresent:(VungleRewarded * _Nonnull)rewarded withError
 
 - (void)rewardedAdDidClose:(VungleRewarded * _Nonnull)rewarded
 {
+    [self.rewardedAdDelegate didCompleteRewardedAdVideo];
```

Comment:
Is this supposed to be called on close, or on 'successful view'? When testing in Native SDK - there is a callback triggered at 80(?) percent that triggers the incentivized report to the `/ri` endpoint. Just want to be sure that this shouldn't be aligned with that.

Reply:
they have a seperate callback called didRewardUserWithReward which will be called when we trigger - (void)rewardedAdDidRewardUser:(VungleRewarded * _Nonnull)rewarded